homeassistant/components/somfy_mylink/config_flow.py

```diff
@@ -125,71 +141,49 @@ async def async_step_init(self, user_input=None):
             return self.async_abort(reason="cannot_connect")
 
         if user_input is not None:
-            self.options[CONF_DEFAULT_REVERSE] = user_input[CONF_DEFAULT_REVERSE]
-
-            entity_id = user_input.get(CONF_ENTITY_ID)
-            if entity_id:
-                return await self.async_step_entity_config(None, entity_id)
+            target_id = user_input.get(CONF_TARGET_ID)
+            if target_id:
+                return await self.async_step_target_config(None, target_id)
 
             return self.async_create_entry(title="", data=self.options)
 
-        data_schema = vol.Schema(
-            {
-                vol.Required(
-                    CONF_DEFAULT_REVERSE,
-                    default=self.options.get(
-                        CONF_DEFAULT_REVERSE, DEFAULT_CONF_DEFAULT_REVERSE
-                    ),
-                ): bool
-            }
-        )
-        data = self.hass.data[DOMAIN][self.config_entry.entry_id]
-        mylink_entity_ids = data[MYLINK_ENTITY_IDS]
-
-        if mylink_entity_ids:
-            entity_dict = {None: None}
-            for entity_id in mylink_entity_ids:
-                name = entity_id
-                state = self.hass.states.get(entity_id)
-                if state:
-                    name = state.attributes.get(ATTR_FRIENDLY_NAME, entity_id)
-                entity_dict[entity_id] = f"{name} ({entity_id})"
-            data_schema = data_schema.extend(
-                {vol.Optional(CONF_ENTITY_ID): vol.In(entity_dict)}
-            )
+        cover_dict = {None: None}
```

Comment:
What does this allow?

Reply:
This allows them to select no cover so they can exit the config flow.